In [7]:
# uncertainty in EFc from circular motion lab

import numpy as np

def Rule4_2(Q,A,m,errA,B,n,errB):
    errQ = Q*np.sqrt((m*errA/A)**2+(n*errB/B)**2)
    return errQ

Q = 78.3

A = .2083
m = 1
errA = .0001

B = .105
n = -1
errB = .0032

errQ = Rule4_2(Q,A,m,errA,B,n,errB)

print('errQ =', errQ)

errQ = 2.3865817646556438


this result is the same as the one I found by hand. Everything works!

In [12]:
x = np.array([ 1.1, 1.3, 1.4, 0.9, 0.95, 1.05])

ave_x = np.average(x)

std_x = np.std(x)

print("ave_x =", ave_x)
print("std_x =", std_x)

ave_x = 1.1166666666666667
std_x = 0.17950549357115014


the average of x is 1.12, the standard deveation of x is .180


$\delta Q = \sqrt{(\delta A)^2 +(\delta B)^2}$



$\delta \Sigma F_{c} = \Sigma F_{c} * \sqrt{(\frac{\delta m}{m})^2+(\frac{-\delta slope}{slope})^2}$